dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05380ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,462 @@
+                                            "\"uuid\": \"1\"" +
+                                        "}" +
+                                    "]," +
+                                    "\"widthPercent\": 83," +
+                                    "\"leftOffset\": 1," +
+                                    "\"width\": 10," +
+                                    "\"left\": 0" +
+                                "}" +
+                            "]" +
+                        "}" +
+                    "]" +
+            "}" +
+        "}";
+
+    final String legacyHTMLLayout =
+            "<div id=\"resp-template\" name=\"globalContainer\">" +
+                "<div id=\"hd-template\"><h1>Header</h1></div>" +
+                "<div id=\"bd-template\">" +
+                    "<div id=\"yui-main-template\">" +
+                        "<div class=\"yui-b-template\" id=\"splitBody0\">" +
+                            "<div class=\"addContainerSpan\">" +
+                                "<a href=\"javascript: showAddContainerDialog('splitBody0');\" title=\"Add Container\">" +
+                                    "<span class=\"plusBlueIcon\"></span>Add Container" +
+                                "</a>" +
+                            "</div>" +
+                            "<span class=\"titleContainerSpan\" id=\"splitBody0_span_69b3d24d-7e80-4be6-b04a-d352d16493ee_1562770692396\" title=\"container_69b3d24d-7e80-4be6-b04a-d352d16493ee\">" +
+                                "<div class=\"removeDiv\">" +
+                                    "<a href=\"javascript: removeDrawedContainer('splitBody0','69b3d24d-7e80-4be6-b04a-d352d16493ee','1562770692396');\" title=\"Remove Container\">" +
+                                        "<span class=\"minusIcon\"></span>Remove Container" +
+                                    "</a>" +
+                                "</div>" +
+                                "<div class=\"clear\"></div>" +
+                                "<h2>Container: Default</h2>" +
+                                "<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>" +
+                            "</span>" +
+                            "<div style=\"display: none;\" title=\"container_69b3d24d-7e80-4be6-b04a-d352d16493ee\" id=\"splitBody0_div_69b3d24d-7e80-4be6-b04a-d352d16493ee_1562770692396\">" +
+                                "#parseContainer('%s/application/containers/default/','1')\n" +
+                            "</div>" +
+                        "</div>" +
+                    "</div>" +
+                "</div>" +
+                "<div id=\"ft-template\">" +
+                    "<h1>Footer</h1>" +
+                "</div>" +
+            "</div>\n";
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)